src/test/isolation2/sql/fts_segment_reset.sql

```diff
@@ -29,6 +29,11 @@ from gp_segment_configuration where role = 'p' and content = 0;
 from gp_segment_configuration where role = 'p' AND content = 0;
 1&:create table fts_reset_t(a int);
 
+-- Make sure that fault is processed and the following query started after seg0 has actually gone
+-- Of course, sleeping is not the best solution, but wait_until_triggered_fault won't work here 
+-- because seg0 process with faults hash table has gone.
+select pg_sleep(0.5);
```

Comment:
Can we use something [like](https://github.com/arenadata/gpdb/commit/fd88a67a8f29cf2d537a73c6c04fc492cb06c736)?

Reply:
I believe cyclic reading of the table with a sleep is not the best solution - it will take much more resources and introduce unneeded complexity. I think wait on injected fault is more suitable here.